### PR TITLE
Mark azure-mgmt-servicelinker as New

### PR DESCRIPTION
**Pull request by danieljurek**
@weshaggard -- I think this track 2 package is not marked correctly in the metadata which prevents it from being onboarded to docs.ms.

### CODE REVIEW

File: _data/releases/latest/python-packages.csv [hunk 1]
```diff
@@ -296,7 +296,7 @@
 "azure-mgmt-resourceconnector","","1.0.0b1","azure-mgmt-resourceconnector","","NA","NA","NA","","false","","","","","",""
 "azure-mgmt-resourcehealth","","1.0.0b1","azure-mgmt-resourcehealth","","NA","NA","NA","","false","","","","","",""
 "azure-mgmt-servicefabricmanagedclusters","1.0.0","","azure-mgmt-servicefabricmanagedclusters","","NA","NA","NA","","false","","","","","",""
-"azure-mgmt-servicelinker","","1.0.0b1","azure-mgmt-servicelinker","","NA","NA","NA","","false","","","","","",""
+"azure-mgmt-servicelinker","","1.0.0b1","azure-mgmt-servicelinker","Service Linker","https://github.com/Azure/azure-sdk-for-python/tree/azure-mgmt-servicelinker_1.0.0b1/sdk/servicelinker/azure-mgmt-servicelinker","NA","NA","","true","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/python-packages.csv:299]:
```suggestion
"azure-mgmt-servicelinker","","1.0.0b1",""Resource Management - Service Linker","Service Linker","servicelinker","NA","NA","mgmt","true","","","","","",""
```

Comment (weshaggard) [reply _data/releases/latest/python-packages.csv]: @danieljurek why not take the repopath update as well?

Comment (danieljurek) [reply _data/releases/latest/python-packages.csv]: They should all be in there now.

File: _data/releases/latest/python-packages.csv [hunk 2]
```diff
@@ -296,7 +296,7 @@
 "azure-mgmt-resourceconnector","","1.0.0b1","azure-mgmt-resourceconnector","","NA","NA","NA","","false","","","","","",""
 "azure-mgmt-resourcehealth","","1.0.0b1","azure-mgmt-resourcehealth","","NA","NA","NA","","false","","","","","",""
 "azure-mgmt-servicefabricmanagedclusters","1.0.0","","azure-mgmt-servicefabricmanagedclusters","","NA","NA","NA","","false","","","","","",""
-"azure-mgmt-servicelinker","","1.0.0b1","azure-mgmt-servicelinker","","NA","NA","NA","","false","","","","","",""
+"azure-mgmt-servicelinker","","1.0.0b1","Resource Management - Service Linker","Service Linker","https://github.com/Azure/azure-sdk-for-python/tree/azure-mgmt-servicelinker_1.0.0b1/sdk/servicelinker/azure-mgmt-servicelinker","NA","NA","mgmt","true","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/python-packages.csv:299]:
```suggestion
"azure-mgmt-servicelinker","","1.0.0b1","Resource Management - Service Linker","Service Linker","servicelinker","NA","NA","mgmt","true","","","","","",""
```

**Review by weshaggard — APPROVED** [review]

**Pull request merged**